### ISSUE TITLE
Allow furniture that can be used to be disconnected, moved, and installed

### ISSUE THREAD

**Issue by PatrikLundell** [CleverRaven/Cataclysm-DDA#58742]
### Is your feature request related to a problem? Please describe.

The "furniture" I'm currently annoyed by is the various types of training equipment that can be used and looks like it should be easy to take down, transport and then install, but the issue as such is not limited to that set, but rather everything that used to just be junk that could be salvaged for parts at best, but now has been given an actual use.

### Solution you would like.

Annoy the scavengers and make the the home decorators happy by changing furniture deconstruction into simple deconstruction tasks that yield the items and move the current deconstruction into deconstruction recipes for the items. Scavengers would then have to deconstruct the items in a two step process (but the second, longer, step could then be handed over to companions, so it's not all negative for the scavengers).

Note that only items that can reasonably be disconnected as items should be converted.

**Comment by Traple** [comment]
I had a look at various recipes. I think deconstruction mostly works. Special mention to the punching bag which currently doesn't have a deconstruction recipe:
https://nornagon.github.io/cdda-guide/#/furniture/f_punching_bag

**Comment by PatrikLundell** [comment]
Yes, I think the main effort would go into creating the appropriate items, with the attendant research into dimensions and weight.
